data-service/server/lib/api.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
```

Comment:
I saw the lib/ directory was already updated in `.gitignore`, is it also worth removing the files from source control?